VATSpy.dat

```diff
@@ -18083,11 +18089,13 @@ MTEG|Port-Au-Prince||MTEG
 MUFH|Havana|HAV|MUFH
 MYNN|Nassau|MYNN|MYNN
 NAT|Shanwick & Gander|NAT|NAT
+NCRG|Rarotonga|NCRG
 NFFF|Nadi Oceanic||NFFF
 NFFN|Nadi||NFFN
 NVVV|Port Vila||NVVV
 NSTU|Pago Pago||NZZO
-NTTT|Tahiti Radio||NTTT
+NTTT|Tahiti||NTTT
+NTTC|Tahiti||NTTC

```

Comment:
These will be called "Tahiti Center" on VATSpy as you have not defined the callsign ending on line 120:

French Polynesia|NT|

Make it 
French Polynesia|NT|Control